src/consensus/pbft/libbyz/Checkpoint.cpp

```diff
@@ -18,23 +18,24 @@ Checkpoint::Checkpoint(Seqno s, Digest& d, bool stable) :
     sizeof(Checkpoint_rep) + pbft::GlobalState::get_node().auth_size())
 {
 #else
-  Message(
-    Checkpoint_tag,
-    sizeof(Checkpoint_rep) + pbft::GlobalState::get_node().sig_size())
+  Message(Checkpoint_tag, sizeof(Checkpoint_rep) + pbft_max_signature_size)
 {
 #endif
   rep().extra = (stable) ? 1 : 0;
   rep().seqno = s;
   rep().digest = d;
   rep().id = pbft::GlobalState::get_node().id();
+#ifdef USE_PKEY_CHECKPOINTS
```

Comment:
That doesn't seem exposed through CMake?

Reply:
No these are hard coded in `parameters.h`. We should make a pass at the file and remove the ifdefs that are not needed anymore and are always set, but not as a part of this PR :)

Reply:
Ok!